From 78054f19bfe20555792b0f336b423921fe88b994 Mon Sep 17 00:00:00 2001
From: Anders Kaseorg <andersk@mit.edu>
Date: Mon, 1 Mar 2021 11:34:23 -0800
Subject: [PATCH] Remove test broken by Python CVE-2021-23336 fix

In Python 3.6.13, 3.7.10, 3.8.8, and 3.9.2, urllib.parse.parse_qsl no
longer treats ; as a separator by default
(https://bugs.python.org/issue42967).

Fixes #164.

Signed-off-by: Anders Kaseorg <andersk@mit.edu>
---
 tests/test_url.py | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/tests/test_url.py b/tests/test_url.py
index 3ff544e..e90065c 100644
--- a/tests/test_url.py
+++ b/tests/test_url.py
@@ -299,10 +299,6 @@ def test_add_or_replace_parameter(self):
         self.assertEqual(add_or_replace_parameter(url, 'arg3', 'nv3'),
                          'http://domain/test?arg1=v1&arg2=v2&arg3=nv3')
 
-        url = 'http://domain/test?arg1=v1;arg2=v2'
-        self.assertEqual(add_or_replace_parameter(url, 'arg1', 'v3'),
-                         'http://domain/test?arg1=v3&arg2=v2')
-
         self.assertEqual(add_or_replace_parameter("http://domain/moreInfo.asp?prodID=", 'prodID', '20'),
                          'http://domain/moreInfo.asp?prodID=20')
         url = 'http://rmc-offers.co.uk/productlist.asp?BCat=2%2C60&CatID=60'
